### ISSUE TITLE
Tinkers crash and some upgrades that I put in the tool disapear when I use the blood chest.

### ISSUE THREAD

**Issue by Legegends** [rubensworks/EvilCraft#532]
I don't know if someone have posted this but here we go.

if you have a tinker tool, with like 

Redstone 29/80 its somehow just disapear if you put in the blood chest.
and have a lot of crash when I put a tinker tool in the tank, if I stay with no inventory or chest open don't crash
I think this is because the chest some times try to put vengeance in the tinker tool. 

and if you put a broken weapon in the blood chest it gonna repair it but the tool will have the stats of the broken tool.
is easy to fix just make the chest don't repair broken tools.
[crash-2017-02-26_12.22.13-server.txt](https://github.com/CyclopsMC/EvilCraft/files/802245/crash-2017-02-26_12.22.13-server.txt)

**Comment by rubensworks** [comment]
@bonii-xx, this crash seems to occur when invoking `ItemStack#copy` on a TCon tool.
Any ideas what could cause this? Perhaps an update to the `ItemStack` from the network/render thread?

**Comment by bonii-xx** [comment]
From the stacktrace it looks like a perfectly save call. Itemstack.copy works fine otherwise too, it's often used in TiC. TiC doesn't use custom packets for tool data, and vanilla only syncs it during the update phase of the world, which is in the same thread as update TE. If I had to guess I'd say something else is interfering with it, try reproducing with only a EvilCraft and TiC/Mantle.

**Comment by rubensworks** [comment]
@bonii-xx Ok, thanks for your input!

Chances are big that another mod is indeed causing is, because I'm not able to reproduce it myself with just EC and TiC. @O-Barbaro-Desenhista Could you try it on your end with just these mods?

**Comment by Legegends** [comment]
I tried 

**First try**
Can't even repair the weapons, but I get a silly graphical bug using the creative blood drop Here:
![2017-03-03_19 42 26](https://cloud.githubusercontent.com/assets/24391537/23571737/a47a55ea-0049-11e7-889c-2d9d01715998.png)

**Second try**
Added TinkersToolLeveling.
And it started to repair the tools(realy don't know how), and no crash and no redstone or buff added to the tool gone or something, and the tool don't lose the status that have when not broken.

**Third try**
Added Tinker I/O.
Nothing happened.

Whep I coud not replicate it too.

So I think is not your mod but a addon.................................. I Know, I know I'm a idiot sorry for this.
